src/Microsoft.OData.Core/UriParser/Visitors/IsCollectionTranslator.cs

```diff
@@ -62,6 +62,18 @@ public override bool Translate(PropertySegment segment)
             return false;
         }
 
+        /// <summary>
+        /// Translate an AnnotationSegment

```

Comment:
<code>AnnotationSegement</code>?

Reply:
Internal class, so not really worth applying change here and (for consistency) in all the other methods.

---
In reply to: [167040541](https://github.com/OData/odata.net/pull/1063#discussion_r167040541) [](ancestors = 167040541)